server/src/loadtest/nearbyGQLQuery.js

```diff
@@ -0,0 +1,53 @@
+import http from 'k6/http'
+import { check } from 'k6'
+let host = `http://${__ENV.HOST || 'localhost:3000'}`
+
+function getMainPageQuery() {
+  let query = `
+  query FetchNearbyMap($loc: LocationInput!) {
+    nearby(loc: $loc) {
+      id
+      createdAt
+      creator {
+        username
+        id
+      }
+      seen
+      name
+      location {
+        lat
+        lng
+      }
+      type
+      uri
+    }
+  }
+  `
+  let params = {
+    headers: {
+      'Content-Type': 'application/json',
+      'User-Agent': 'robert/k6',
```

Comment:
Long term, this isn't something I would commit to the repository. Here, it's fine though.